SPECS/python-rpm-spec/python-rpm-spec.signatures.json

```
@@ -0,0 +1,5 @@
+{
+ "Signatures": {
+  "python-rpm-spec-v0.14.1.tar.gz": "7256434000b9192ace14a51a05a58503f1982163beddcf46ddfa0a8553c11570"
```

Thank you for updating the spec's source to `%{name}-%{version}.tar.gz`. We missed updating the signature files along with it:
```suggestion
  "python-rpm-spec-0.14.1.tar.gz": "7256434000b9192ace14a51a05a58503f1982163beddcf46ddfa0a8553c11570"
```